openai/openai.go

```diff
@@ -357,7 +400,37 @@ func ListMiddleware() gin.HandlerFunc {
 	}
 }
 
-func Middleware() gin.HandlerFunc {
+func DeleteMiddleware() gin.HandlerFunc {
```

Comment:
In this case I don't think you want middleware as it's an outright handler

Reply:
Ah wait, this is for Delete, sorry. I think it might be worth making this one a handler as well